doc/_extensions/zephyr/kconfig/static/kconfig.mjs

```diff
@@ -5,6 +5,7 @@
 
 const DB_FILE = 'kconfig.json';
 const RESULTS_PER_PAGE_OPTIONS = [10, 25, 50];
+const ZEPHYR_GH_BASE_URL = 'https://github.com/zephyrproject-rtos/zephyr';
```

Comment:
this needs to be configurable somehow, via conf.py

Reply:
Agreed - is there another example of a javascript file using a var from the conf? Have not messed around with sphinx that much.

Reply:
maybe `kconfig.json` could be used for that?

Reply:
Neat idea, done.